docs/designers-developers/developers/block-api/block-templates.md

```diff
@@ -17,18 +17,51 @@ Planned additions:
 
 Templates can be declared in JS or in PHP as an array of blockTypes (block name and optional attributes).
 
+The first example in PHP creates a template for posts that includes an image block to start, you can add as many or as few blocks to your template as needed.
+
+
+PHP example:
+
+```php
+<?php
+function myguten_register_template() {
```

Comment:
A very similar example can be found in the section at the bottom of the document:

https://github.com/WordPress/gutenberg/blob/58fc19120bbfd74a049d5863c6bcf08e15e1b7e7/docs/designers-developers/developers/block-api/block-templates.md#existing-post-types

It would be nice to consolidate it.

Reply:
Agreed, I combined the example lower in the page with the locking piece above it, so it shows how to set the `template_lock` attribute.